docs/tools/Deploy-IRIS-Monitor.md

```diff
@@ -1,62 +1,56 @@
-# How to deploy IRIShub Monitor
+# How to deploy IRIS Monitor
 
 Please make sure that iris is installed in your computer and added to $PATH.You can see this page for insturcion https://github.com/irisnet/irishub. You also need /bin/bash, wc, ps to ensure the monitor work properly.
```

Comment:
Please refer to the [document](https://github.com/irisnet/irishub/blob/master/docs/get-started/Install-Iris.md) for deatiled instruction.